pkg/controllers/deprovisioning/suite_test.go

```diff
@@ -588,6 +616,60 @@ var _ = Describe("Replace Nodes", func() {
 		Expect(cloudProvider.CreateCalls).To(HaveLen(0))
 		ExpectNodeExists(ctx, env.Client, node.Name)
 	})
+	It("replaces node for expiration", func() {
```

Comment:
nit: for consistent naming

```suggestion
	It("can replace node for expiration", func() {
```